source/containers.tex

```diff
@@ -15032,3 +15034,2422 @@
+\begin{itemize}
+\item
+\tcode{A} models \tcode{copyable},
+\item
+\tcode{is_nothrow_move_constructible_v<A>} is \tcode{true},
+\item
+\tcode{is_nothrow_move_assignable_v<A>} is \tcode{true},
+\item
+\tcode{is_nothrow_swappable_v<A>} is \tcode{true}, and
+\item
+the following types and expressions
+are well-formed and have the specified semantics.
+\end{itemize}
+
+\begin{itemdecl}
+typename A::element_type
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\result
+A complete object type that is not an abstract class type.
+\end{itemdescr}
+
+\begin{itemdecl}
+typename A::pointer
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\result
+A type that models \tcode{copyable}, and
+for which \tcode{is_nothrow_move_constructible_v<A::poin\-ter>} is \tcode{true},
+\tcode{is_nothrow_move_assignable_v<A::pointer>} is \tcode{true}, and
+\tcode{is_nothrow_swappable_v<A::pointer>} is \tcode{true}.
+\begin{note}
+The type of \tcode{pointer} need not be \tcode{element_type*}.
+\end{note}
+\end{itemdescr}
+
+\begin{itemdecl}
+typename A::reference
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\result
+A type that models
+\tcode{common_reference_with<A::reference\&\&, A::element_type\&>}.
+\begin{note}
+The type of \tcode{reference} need not be \tcode{element_type\&}.
+\end{note}
+\end{itemdescr}
+
+\begin{itemdecl}
+typename A::offset_policy
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\result
+A type \tcode{OP} such that:
+
+\begin{itemize}
+\item
+\tcode{OP} meets the accessor policy requirements,
+\item
+\tcode{constructible_from<OP, const A\&>} is modeled, and
+\item
+\tcode{is_same_v<typename OP::element_type, typename A::element_type>} is \tcode{true}.
+\end{itemize}
+\end{itemdescr}
+
+\begin{itemdecl}
+a.access(p, i)
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\result
+\tcode{A::reference}
+
+\pnum
+\remarks
+The expression is equality preserving.
+
+\pnum
+\begin{note}
+Concrete accessor policies can impose preconditions for their access function.
+However, they might not.
+For example, an accessor where
+\tcode{p} is \tcode{span<A::element_type, dynamic_extent>} and
+\tcode{access(p, i)} returns \tcode{p[i \% p.size()]}
+does not need to impose a precondition on \tcode{i}.
+\end{note}
+\end{itemdescr}
+
+\begin{itemdecl}
+a.offset(p, i)
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\result
+\tcode{A::offset_policy::pointer}
+
+\pnum
+\returns
+\tcode{q} such that for \tcode{b} being \tcode{A::offset_policy(a)}, and
+any integer $n$ for which $[0, n)$ is
+an accessible range of \tcode{p} and \tcode{a}:
+\begin{itemize}
+\item
+$[0, n - \tcode{i})$ is an accessible range of \tcode{q} and \tcode{b}; and
+\item
+\tcode{b.access(q, $j$)} provides access to
+the same element as \tcode{a.access(p, i + $j$)},
+for every $j$ in the range $[0, n - \tcode{i})$.
+\end{itemize}
+
+\pnum
+\remarks
+The expression is equality-preserving.
+\end{itemdescr}
+
+\rSec3[mdspan.accessor.default]{Class template \tcode{default_accessor}}
```

Comment:
This is short enough that its child subclauses could be collapsed. What do you think?